app/Models/Season.php

```diff
@@ -45,6 +48,36 @@ public static function latestOrId($id)
         return $season;
     }
 
+    public function divisions(): HasMany
+    {
+        return $this->hasMany(Division::class);
+    }
+
+    public function divisionsOrdered(): Collection
+    {
+        return cache_remember_mutexed(
+            "divisions:{$this->id}",
+            $GLOBALS['cfg']['osu']['seasons']['divisions_cache_duration'],
```

Comment:
saw duration being pulled from config/env in other places (tags) recently so just mimicked that, but wondering if this is actually needed here to begin with :thinking:

Reply:
no it's not. the other place involves some aggregation and stuff